demo/lightning/image-segmentation/dataset.py

```diff
@@ -31,99 +30,68 @@ def __init__(
         self,
         project_name,
         bucket_name,
-        config=dataflux_mapstyle_dataset.Config(),
         storage_client=None,
         **kwargs,
     ):
 
         self.train_transforms = kwargs["transforms"]
         patch_size, oversampling = kwargs["patch_size"], kwargs["oversampling"]
-        images_prefix, labels_prefix = kwargs["images_prefix"], kwargs[
-            "labels_prefix"]
         self.patch_size = patch_size
         self.rand_crop = RandBalancedCrop(patch_size=patch_size,
                                           oversampling=oversampling)
 
         # Dataflux-specific setup.
         self.project_name = project_name
         self.bucket_name = bucket_name
-        self.config = config
-        self.dataflux_download_optimization_params = (
-            dataflux_core.download.DataFluxDownloadOptimizationParams(
-                max_composite_object_size=self.config.max_composite_object_size
-            ))
-        if not storage_client:
-            self.storage_client = storage.Client(
-                project=project_name,
-            )
-
-        # Data listing.
-        self.images = dataflux_core.fast_list.ListingController(
-            max_parallelism=self.config.num_processes,
-            project=self.project_name,
-            bucket=self.bucket_name,
-            sort_results=self.config.
-            sort_listing_results,  # This needs to be True to map images with labels.
-            prefix=images_prefix,
-        ).run()
-        self.labels = dataflux_core.fast_list.ListingController(
-            max_parallelism=self.config.num_processes,
-            project=self.project_name,
-            bucket=self.bucket_name,
-            sort_results=self.config.
-            sort_listing_results,  # This needs to be True to map images with labels.
-            prefix=labels_prefix,
-        ).run()
+
+        self.images_in_bytes = dataflux_mapstyle_dataset.DataFluxMapStyleDataset(
```

Comment:
nit: might be better named with something like "images_dataset" because we used "images_in_bytes" to represent the downloaded images files in the format of bytes. Here we are just defining a dataset and haven't downloaded anything yet.